### PR TITLE
Rename store to place

### PR DESCRIPTION
**Pull request by nicksellen**
Reworking of https://github.com/yunity/karrot-backend/pull/585 on top of current master.

### CODE REVIEW

File: foodsaving/management/commands/influxutil.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                '-compress',
+            ])
+        elif command == 'import':
+            dump = argv.pop(0)
+            execute([
+                'influx',
+                *influx_args,
+                '-import',
+                '-database',
+                'karrot-foo',
```
Comment (tiltec) [inline foodsaving/management/commands/influxutil.py]: karrot-foo?

File: foodsaving/notifications/models.py [hunk 2]
```diff
@@ -17,7 +17,7 @@ class NotificationType(Enum):
     USER_BECAME_EDITOR = 'user_became_editor'
     YOU_BECAME_EDITOR = 'you_became_editor'
     FEEDBACK_POSSIBLE = 'feedback_possible'
-    NEW_STORE = 'new_store'
+    NEW_STORE = 'new_place'
```
Comment (tiltec) [inline foodsaving/notifications/models.py]: Could be left as `new_store` as type-specific notification

File: foodsaving/pickups/serializers.py [hunk 3]
```diff
@@ -257,24 +257,24 @@ def create(self, validated_data):
...
-    def validate_store(self, store):
-        if not store.group.is_editor(self.context['request'].user):
+    def validate_place(self, place):
+        if not place.group.is_editor(self.context['request'].user):
             raise PermissionDenied(_('You need to be a group editor'))
-        if not store.group.is_member(self.context['request'].user):
-            raise serializers.ValidationError(_('You are not member of the store\'s group.'))
-        return store
+        if not place.group.is_member(self.context['request'].user):
+            raise serializers.ValidationError(_('You are not member of the place\'s group.'))
```
Comment (tiltec) [inline foodsaving/pickups/serializers.py]: Translation message

File: foodsaving/pickups/serializers.py [hunk 4]
```diff
@@ -343,22 +343,22 @@ class Meta:
...
     def get_is_editable(self, feedback):
         return feedback.about.is_recent() and feedback.given_by_id == self.context['request'].user.id
 
     def validate_about(self, about):
         user = self.context['request'].user
-        group = about.store.group
+        group = about.place.group
         if not group.is_member(user):
-            raise serializers.ValidationError(_('You are not member of the store\'s group.'))
+            raise serializers.ValidationError(_('You are not member of the place\'s group.'))
```
Comment (tiltec) [inline foodsaving/pickups/serializers.py]: Translation message

File: foodsaving/pickups/tests/test_feedback_api.py [hunk 5]
```diff
@@ -87,12 +87,12 @@ def test_create_feedback_fails_as_non_user(self):
...
     def test_create_feedback_fails_as_non_group_member(self):
         """
-        User is not allowed to give feedback when not a member of the store's group.
+        User is not allowed to give feedback when not a member of the place's group.
         """
         self.client.force_login(user=self.user)
         response = self.client.post(self.url, self.feedback_post, format='json')
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST, response.data)
-        self.assertEqual(response.data, {'about': ['You are not member of the store\'s group.']})
+        self.assertEqual(response.data, {'about': ['You are not member of the place\'s group.']})
```
Comment (tiltec) [inline foodsaving/pickups/tests/test_feedback_api.py]: This should fail when the translation message stays the same

**Comment by tiltec** [comment]
FYI, I cherry-picked "Don't fail if cannot register CustomDateTimeTZRange" to master

File: foodsaving/notifications/models.py (hunk 2 above)
Comment (nicksellen) [reply foodsaving/notifications/models.py]: I went with `NEW_STORE` --> `NEW_PLACE`

File: foodsaving/pickups/serializers.py (hunk 3 above)
Comment (nicksellen) [reply foodsaving/pickups/serializers.py]: I left these ones in as "place" actually. They are these kind of error messages that should not normally be seen...

**Pull request merged**